exercises/concept/shopping/.docs/introduction.md

```diff
@@ -30,7 +30,7 @@ In the examples below you'll encounter:
     `sed` and `awk` both use the `-f` option for the same purpose.
     You will see this used in the test scripts for the practice exercises.
 
-The rest of this lesson will focus on the `jq` _lanaguage_.
+The rest of this lesson will focus on the `jq` _language_.
```

Comment:
Thanks for the catch! Did you run the configlet tool? This file is auto-generated from `concepts/basics/introduction.md`

Reply:
No, i just submitted the change through github's web UI, I don't have a copy of the repo on my machine so I cant run the script. I'll go and change the basics/introduction.md file